app/src/androidTest/java/org/kiwix/kiwixmobile/NetworkTest.kt

```diff
@@ -109,6 +109,7 @@ class NetworkTest {
     try {
       Espresso.onView(ViewMatchers.withId(android.R.id.button1)).perform(ViewActions.click())
     } catch (e: RuntimeException) {
+      e.printStackTrace()
```

Comment:
It would be beneficial to provide a clear error message that is more contextually relevant.
Something like,

```
  Log.e(NETWORK_TEST_TAG, "Failed to find button 1 of alert dialog, original exception is ${e.printStackTrace()}")